src/components/Board.js

```diff
@@ -5,22 +5,159 @@ import axios from 'axios';
 import './Board.css';
 import Card from './Card';
 import NewCardForm from './NewCardForm';
-import CARD_DATA from '../data/card-data.json';
+// import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
   constructor() {
     super();
 
     this.state = {
       cards: [],
+      errors: [],
+      ids: [],
     };
+  };
+
+  componentDidMount() {
+    const GET_ALL_CARDS_URL = "https://inspiration-board.herokuapp.com/boards/redPanda/cards";
+
+    axios.get(GET_ALL_CARDS_URL)
+    .then((response) => {
+      this.setState({
+        errors: []
+      });
+
+      let updatedIDsAfterGet = response.data.map((card) => {
+        // console.log(card.card.id);
+        return card.card.id
+      })
+
+      this.setState({
+        cards: response.data,
+        ids: updatedIDsAfterGet,
+      });
```

Comment:
Do you need to keep track of both the cards and the IDs separately? Having parallel arrays like this always makes me a little nervous, since it means anywhere you touch one you have to remember to update the other, making it more likely for future you to make a mistake.

I also don't think you're ever reading this array anywhere, so you could probably remove it from the state without too much trouble.